common/rfb/VNCSConnectionST.h

```diff
@@ -105,6 +105,11 @@ namespace rfb {
     // or because the current cursor position has not been set by this client.
     bool needRenderedCursor();
 
+    // clientInitReady() is called when the desktop has finished
+    // initializing and is ready for a client.
+    void clientInitReady();
```

Comment:
I would suggest keep the naming focused on what happened. I.e. "desktopReady()" here as well.